File: store_agent/src/main/java/com.abc/dao/ScoreDAOImpl.java
package com.abc.dao;

import com.abc.model.VendorResponse;
import com.abc.model.rest.DecisionResponse;
import com.abc.model.rest.UserRequest;
import org.springframework.stereotype.Repository;

@Repository
public class ScoreDAOImpl implements ScoreDAO {

    public void saveUserRequest(UserRequest request) {
        System.out.println("saveUserRequest");
    }

    public void storeVendorResponse(VendorResponse[] vendorResponse) {
        System.out.println("storeVendorResponse");
    }

    public void storeDecision(DecisionResponse decision) {
        System.out.println("storeDecision");
    }
}

File: store_agent/src/main/java/com.abc/dao/ScoreDAOImpl.kt
package com.abc.dao

import com.abc.model.VendorResponse
import com.abc.model.rest.DecisionResponse
import com.abc.model.rest.UserRequest
import org.springframework.stereotype.Repository

@Repository
class ScoreDAOImpl : ScoreDAO {
    override fun saveUserRequest(request: UserRequest) {
        println("saveUserRequest: $request")
    }

    override fun storeVendorResponse(vendorResponse: Array<VendorResponse>) {
        println("storeVendorResponse: $vendorResponse")
    }

    override fun storeDecision(decision: DecisionResponse) {
        println("storeDecision: $decision")
    }
}